components/HistoricalWithdrawal.tsx

```diff
@@ -0,0 +1,180 @@
+import { Dispatch, SetStateAction, useEffect, VFC } from "react";
+import { IntrinsicElements } from "@/types";
+import { css } from "@emotion/react";
+import {
+	Accordion,
+	AccordionDetails,
+	AccordionSummary,
+	TextField,
+	Theme,
+	Tooltip,
+} from "@mui/material";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
+import { useBridge } from "@/providers/BridgeProvider";
+
+interface HistoricalWithdrawalProps {
+	expanded: boolean;
+	setExpanded: Dispatch<SetStateAction<boolean>>;
+}
+
+const HistoricalWithdrawal: VFC<
+	IntrinsicElements["div"] & HistoricalWithdrawalProps
+> = ({ expanded, setExpanded, ...props }) => {
+	const {
+		historicalBlockHash,
+		setHistoricalBlockHash,
+		historicalEventProofId,
+		setHistoricalEventProofId,
+	} = useBridge();
+
+	useEffect(() => {
+		if (expanded) return;
+
+		setHistoricalBlockHash(null);
+		setHistoricalEventProofId(null);
+	}, [expanded, setHistoricalBlockHash, setHistoricalEventProofId]);
+
+	return (
+		<div {...props} css={styles.root}>
+			<Accordion
+				css={[styles.formSettings]}
+				expanded={expanded}
+				onChange={() => setExpanded(!expanded)}
+			>
+				<AccordionSummary
+					css={styles.formSettingsSummary}
+					expandIcon={<ExpandMore />}
+				>
+					Advanced
+				</AccordionSummary>
+				<AccordionDetails>
+					<div css={styles.formField}>
+						<label htmlFor="historicalEventProofId">
+							Historical Event Proof Id
+						</label>
+						<TextField
+							type="number"
+							css={styles.proofId}
+							required={expanded}
+							value={historicalEventProofId ?? 0}
```

Comment:
should be `historicalEventProofId ?? ""`, and you can add a `min="1"` as prop to avoid silent submit with standard `processWithdrawRequest()` because of this condition

```
if (historicalBlockHash && historicalEventProofId > 0)
```